environment/interpretation/lane_detector/data/Samples.model

```diff
@@ -1,42 +1,35 @@
-																											svm_type c_svc
+svm_type c_svc
 kernel_type polynomial
-degree 4
+degree 3
 gamma 0.00520833
 coef0 0
 nr_class 2
-total_sv 31
-rho -0.792476
+total_sv 24
+rho -0.941211
```

Comment:
@abinashmeher999 These are the changes that have occurred after training a model. Instead of making of a new file, he overwritten on a previous one and hence the changes. Same goes for other as well.